### PR TITLE
Correct DNSPOD

### PR DESCRIPTION
**Pull request by NakornC**
Update to the latest IP
https://dns.pub/?lang=en

### CODE REVIEW

**Comment by NakornC** [comment]
@paulmillr 
dns.pub is used to promote their public DNS servers, just like Cloudflare uses https://1.1.1.1
Domains dns.pub and dnspod.cn both belong to DNSPod, Tencent's subsidiary.

**Comment by paulmillr** [comment]
@NakornC can you provide any proofs for your claim? This is an extremely sensitive repository, so we need more than just words.

**Comment by NakornC** [comment]
@paulmillr Sorry for the outdated info, https://dns.pub/ has redirected to https://www.dnspod.cn/Products/publicdns now

The link below is the English version
https://www.dnspod.cn/Products/publicdns?lang=en

**Comment by NakornC** [comment]
DNS over HTTPS
https://doh.pub/dns-query
https://1.12.12.12/dns-query
https://120.53.53.53/dns-query

DNS over TLS
dot.pub
1.12.12.12
120.53.53.53

BTW you probably get 162.14.21.178 & 162.14.21.56 if nslookup with a non-China IP

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Check out the article for more info: [paulmillr.com/posts/encrypted-dns/](https:
...
-| DNSPod                    | 🇨🇳      | Yes          | [Operated](https://docs.dnspod.cn/public-dns/5fb5db1462110a2b153a77dd/) in mainland China                                               | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-https.mobileconfig),  [TLS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-tls.mobileconfig)                                                                                                                            |
+| DNSPod                    | 🇨🇳      | Yes          | [Operated](https://dns.pub/?lang=en) by Tencent in Mainland China                                               | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-https.mobileconfig),  [TLS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-tls.mobileconfig)                                                                                                                            |
```
Comment (paulmillr) [inline README.md:29]: can you change this to https://www.dnspod.cn/Products/publicdns?lang=en

Comment (NakornC) [reply README.md]: all set

**Pull request merged**